index.html

```diff
@@ -41,18 +41,22 @@
     <div class="container-fluid">
         <div class="row">
             <div class="col-md-12">
-                <div class="page-header">
+                <div class="page-header text-center">
                     <h1>Solr-Heatmap-Client based on AngularJS and OL3</h1>
                 </div>
                 <div class="row">
-                    <div class="col-md-12">
-                        <ng-include src="'./app/view/toolbarSearchField.html'"></ng-include>
-                    </div>
+                  <div class="col-md-2"></div>
```

Comment:
Yes, I use this `div` and the analog  empty`div` below to get offset for the search field (it should be a bit narrow as a map). 
S. example:
![bildschirmfoto vom 2016-06-08 15 34 51](https://cloud.githubusercontent.com/assets/3939355/15896036/b5c28ab8-2d8e-11e6-955a-0145c1779c09.png)